service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/AllergyStructureMapperTest.java

```diff
@@ -47,6 +49,12 @@
     private static final String INPUT_JSON_WITH_NO_CATEGORY = TEST_FILE_DIRECTORY + "example-allergy-intolerance-resource-12.json";
     private static final String INPUT_JSON_WITH_UNSUPPORTED_CATEGORY = TEST_FILE_DIRECTORY + "example-allergy-intolerance-resource-13.json";
     private static final String INPUT_JSON_WITH_NO_ASSERTED_DATE = TEST_FILE_DIRECTORY + "example-allergy-intolerance-resource-14.json";
+    private static final String INPUT_JSON_WITH_RELATION_TO_CONDITION_WITH_ONE_NOTE = TEST_FILE_DIRECTORY
+        + "example-allergy-intolerance-resource-15.json";
```

Comment:
Rather than increment these test data files, can we rename them to something better?
Example, for this one, `example-allergy-intolerance-resource-15.json` -> `allergy-intolerance-resource-with-relation-to-condition-with-one-note.json`